assets/Script/technology/techGame1.js

```diff
@@ -0,0 +1,75 @@
+const { changeScene } = require('../util/sceneUtils');
+const { MENU_SCENE } = require('../constants');
+const MAX_BLOCK_NUMBER = 6;
+
+const generateNumberBlocks = () => {
+    return Math.floor(Math.random() * MAX_BLOCK_NUMBER);
+};
+
+const setupEventHandlers = (that) => {
+  that.homeSprite.on('mousedown', () => {
+    changeScene(MENU_SCENE);
+  });
+};
+
+cc.Class({
+    extends: cc.Component,
+
+    properties: {
+        speed : 200
+    },
+
+    onLoad () {
+        this.homeSprite = this.node.getChildByName('home_button');
+        this.blockSprite = this.node.getChildByName('block')
+
+        var numberBlocks = generateNumberBlocks();
+        var blockPosY = this.blockSprite.getPosition().y;
+        var blockWidth = this.blockSprite.getBoundingBox().size.width;   
+        
+        var distanceTwoBlocks = 30;
+
+        //total widths of all blocks
+        var totalWidth = blockWidth * numberBlocks  + distanceTwoBlocks * (numberBlocks - 1)
+        var blockPosX = - totalWidth / 2;

```

Comment:
What is the `-` for here?

Reply:
The middle position of X-coordinate is 0. I wanted to move the first block to the most left of the screen so I put the negative sign (-) to indicate negative X-coordinate.